packages/advjs/src/client/components/internals/AdvCanvas.vue

```diff
@@ -1,20 +1,20 @@
 <script lang="ts" setup>
 import * as BABYLON from '@babylonjs/core'
-import { isClient } from '@advjs/shared/utils'
-import { setup } from '~/setup/babylon'
+// import { isClient } from '@advjs/shared/utils'
+// import { setup } from '~/setup/babylon'
 import { adv } from '~/setup/adv'
 import { useBabylonStore } from '~/stores/babylon'
 
 const bStore = useBabylonStore()
 
 const babylonCanvas = ref()
 
-onMounted(async () => {
-  if (!isClient)
-    return
-  const instance = await setup(babylonCanvas.value)
-  bStore.setInstance(instance)
-})
+// onMounted(async () => {
```

Comment:
I don't think you should comment it directly, we already use the parent component's if to determine whether to show it or not.